scripts/fast-reboot

```diff
@@ -532,7 +532,7 @@ case "$REBOOT_TYPE" in
         check_warm_restart_in_progress
         BOOT_TYPE_ARG=$REBOOT_TYPE
         trap clear_boot EXIT HUP INT QUIT TERM KILL ABRT ALRM
-        sonic-db-cli STATE_DB SET "FAST_REBOOT|system" "1" "EX" "210" &>/dev/null
+        sonic-db-cli STATE_DB SET "FAST_REBOOT|system" "enable" &>/dev/null
```

Comment:
Regarding the naming of system wide flag, please check-- https://github.com/sonic-net/sonic-sairedis/pull/1196#discussion_r1099245793